### ISSUE TITLE
Error when trying to install Monterey

### ISSUE THREAD

**Issue by theholyjosh1** [balopez83/Surface_Pro_3_Hackintosh#23]
I'm doing a fresh install from windows on the latest EFI trying to install Monterey. When clicking install macOS Monterey, it comes up with the error: 'installation requires downloading important content. That content can't be downloaded at this time. Try again later.'

I done some googling and found out this issue was happening to someone else (on a physical MacBook) which was the late 2013 MacBook Pro. MacBookPro11,1 is the late 2013 MacBook Pro. In order to install sonoma, do I have to install an even older version than Monterey then update all the way to Sonoma?

Specs:
Core i7
8gb ram
512gb storage
HD 5000

**Comment by balopez83** [comment]
@ShibeGG1  If you're using a full installer and not just a recovery then it shouldn't be asking that. That said, you don't need to use such an old installer to upgrade. I wouldn't use anything older than Catalina or even Big Sur. Monterey can be installed but is not considered supported at all as certain components driven by custom kexts only work from Catalina and on. This might also be why Monterey is asking to go on the web if you are using the full installer. 

In fact you can directly install Sonoma with the latest EFI though the only reason I don't recommend it is purely because it's easier to upgrade with OCLP already installed

**Comment by balopez83** [comment]
@ShibeGG1 It sounds like you are using a web recovery or that your installer might be damaged. 

Catalina 10.15 and up are fully supported and should install as normal. I have a full installer of all the installers listed as compatible and tested all and they should install properly. So I do suspect either a web recovery or defective installer. 

If you have an Ethernet adapter compatible with macOS it would probably allow you to hardwire connect and do a web install. 

If you are 100% sure that your installer is good you may want to try with a different version. Catalina and Big Sur work well and you can still direct upgrade to Sonoma 

(Deleted previous message because I made a mistake and incorrectly responded with wrong information.)

**Comment by theholyjosh1** [comment]
I am using a web recovery with a usb Ethernet adapter, so I will re-download it to see if I get any different results. I will update back soon. Thanks for your quick response! 🙃

**Comment by theholyjosh1** [comment]
I re-downloaded macOS Monterey and it gave out the same error. Tried big sur, and from the opencore boot menu and i select the USB it doesn't even get to the Apple logo its just black and then comes with the error: 'OCB: LoadImage failed - Unsupported'
(these are both web installers so i have yet to try a non web installer.)

**Comment by balopez83** [comment]
@ShibeGG1 Honestly web installers are not really well supported on Surface devices that don't use Intel WiFi cards. There are too many variables with usb Ethernet adapters and no appropriate drivers that can be included in the EFI to support most of them. With an unsupported WiFi card on this Surface it causes problems with the web installers.

I have only tested and officially support the full installers.

I realized that the QuickStart guide provided links to creating the web installer which was carried over from another repo that was designed for Intel WiFi cards. I have updated the links to show how to make a full installer using transmac. Apologies for missing this error.

One user in my SP4 repo does use the web installers successfully but does so using a phone connected via USB and tethering. It should work with both iPhone and Android.
